src/mds/MDSRank.cc

```diff
@@ -1930,6 +1930,17 @@ void MDSRank::resolve_done()
   snapclient->sync(new C_MDSInternalNoop);
 }
 
+static size_t apply_blocklist(MDSRank *mds, Server *server,
+                              const std::set<entity_addr_t> &addrs, epoch_t epoch) {
+  auto victims = server->apply_blocklist(addrs);
+  if (victims) {
+    mds->set_osd_epoch_barrier(epoch);
+  }
+
```

Comment:
I would think it'd be nice to have consistent debug output when applying blocklists from the OSDMap. So just move the debug messages below here as one consistent message.